OmniKit/MessageTransport/MessageBlocks/PodInfoActivationTime.swift

```diff
@@ -8,23 +8,26 @@
 
 import Foundation
 
-public struct PodInfoFault : PodInfo {
+// Type 5 PodInfo returns the pod activation time, time pod alive, and the possible fault code
+public struct PodInfoActivationTime : PodInfo {
     // OFF 1  2  3  4 5  6 7 8 9 10111213 1415161718
     // DATA   0  1  2 3  4 5 6 7 8 9 1011 1213141516
     // 02 11 05 PP QQQQ 00000000 00000000 MMDDYYHHMM
 
-    public var podInfoType: PodInfoResponseSubType = .fault
+    public let podInfoType: PodInfoResponseSubType = .activationTime
     public let faultEventCode: FaultEventCode
     public let timeActivation: TimeInterval
     public let dateTime: DateComponents
     public let data: Data
     
     public init(encodedData: Data) throws {
         
-        if encodedData.count < 16 {
+        guard let podInfoType = PodInfoResponseSubType(rawValue: encodedData[0]), podInfoType == self.podInfoType else {
+            throw MessageError.unknownValue(value: encodedData[0], typeDescription: "activationTime")
```

Comment:
```suggestion
            throw MessageError.unknownValue(value: encodedData[0], typeDescription: "PodInfoResponseSubType")
```

Reply:
MessageError.unknownValue is used to report unknown values for raw enum types, and the typeDescription should describe which type the value is unknown for.

I believe this particular value is already checked earlier in the code, so I don't think this will ever be thrown, and could be removed. There is a lot of this checking added in this PR.

There are two things being checked here, in the two clauses of the guard statement: 1) that encodedData[0] is a valid raw value for PodInfoResponseSubType, 2) that the decoded PodInfoResponseSubType is a .activationTime instance. The MessageError.unknownValue error is appropriate for #1, but not #2.  It might be simpler to just do:

```
guard encodedData[0] == self.podInfoType.rawValue else {
```

But even simpler would be to just remove this check (and the similar checks below), as I stated above, as I believe it's already assured earlier in the code.